presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestOrcStorageManager.java

```diff
@@ -219,6 +229,36 @@ public void testWriter()
             assertEquals(stringVector.vector[0], utf8Slice("hello"));
             assertEquals(stringVector.vector[1], utf8Slice("bye"));
 
+            SingleObjectVector arrayVector = new SingleObjectVector();
```

Comment:
These tests seem to be duplicated in `TestShardWriter`. I think we can leave this alone. The purpose of this test class is to test the storage manager end-to-end.